callback-free-itx/script.ts

```diff
@@ -69,17 +71,18 @@ async function main() {
   const tx = await prisma.$begin();
   const user = await tx.user.findFirstOrThrow();
 
-  const tx2 = await prisma.$begin();
-  await tx2.user.findMany();
+  // const tx2 = await prisma.$begin();
+  // await tx2.user.findMany();
```

Comment:
to me the important part to show in transactions is rollback, maybe we should show both.